### DISCUSSION TITLE
更新2.22.1了之后fatal: No remote configured to list refs from.回退2.22.0也没解决

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#142] [问题求助] [answered]
![image](https://user-images.githubusercontent.com/66856838/107952321-947e7f00-6fd4-11eb-809f-8c52a7645d9b.png)
上海阿里云和香港腾讯云都出现了这个问题
用是能用，但是会把服务器CPU吃满

**Comment by Tsuk1ko** [comment, marked as answer]
这确实是个 bug，已经修复，更新到最新版再试试
